src/util/notifications.tsx

```diff
@@ -0,0 +1,19 @@
+import { ICONS } from "@canonical/react-components";
+
+export type Filters = {
+  [filter: string]: boolean;
```

Comment:
I have adjusted the type for `Filters`. I think having filters as an object allows for faster filtering of notifications since we don't need to iterate the array of filters for each notification. Also when adding/removing filters, we wound't have to iterate the array of filters each time.

Reply:
Maybe a better approach is to use a `new Set([])` then? So we have an ordered array with constant time lookup and simple add/remove/isEmpty functions in the interface.

Reply:
good idea, I should use `Set` more!